xarray/tests/test_cftimeindex.py

```diff
@@ -132,16 +134,26 @@ def date_dict(
     list(ISO8601_LIKE_STRING_TESTS.values()),
     ids=list(ISO8601_LIKE_STRING_TESTS.keys()),
 )
-def test_parse_iso8601_like(string, expected):
-    result = parse_iso8601_like(string)
+@pytest.mark.parametrize("five", [False, True], ids=["4Y", "5Y"])
```

Comment:
```suggestion
@pytest.mark.parametrize("five-digit-year", [False, True], ids=["4Y", "5Y"])
```

took me a bit to understand what this meant :)

Reply:
Yeah, I was just taking what came first in my mind...